plugins/modules/packaging/language/ansible_galaxy_install.py

```diff
@@ -0,0 +1,316 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# (c) 2021, Alexei Znamensky <russoz@gmail.com>
+#
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+DOCUMENTATION = """
+module: ansible_galaxy_install
+author:
+    - "Alexei Znamensky (@russoz)"
+short_description: Install Ansible roles or collections using ansible-galaxy
+version_added: 3.5.0
+description:
+  - This module allows the installation of Ansible collections or roles using C(ansible-galaxy).
+notes:
+  - >
+    B(Ansible 2.9/2.10): The C(ansible-galaxy) command changed significantly between Ansible 2.9 and
+    ansible-base 2.10 (later ansible-core 2.11). See comments in the parameters.
+options:
+  type:
+    description:
+    - The type of installation performed by C(ansible-galaxy).
+    - If I(type) is C(both), then I(requirements_file) must be passed and it may contain both roles and collections.
+    - "Note however that the opposite is not true: if using a I(requirements_file), then I(type) can be any of the three choices."
+    - "B(Ansible 2.9): The option C(both) will have the same effect as C(role)."
+    type: str
+    choices: [collection, role, both]
+    required: true
+  name:
+    description:
+    - Name of the collection or role being installed.
+    - Versions can be specified with C(ansible-galaxy) usual formats. For example, C(community.docker:1.6.1) or C(ansistrano.deploy,3.8.0).
+    - I(name) and I(requirements_file) are mutually exclusive.
+    type: str
+  requirements_file:
+    description:
+    - Path to a file containing a list of requirements to be installed.
+    - It works for I(type) equals to C(collection) and C(role).
+    - I(name) and I(requirements_file) are mutually exclusive.
+    - "B(Ansible 2.9): It can only be used to install either a I(type=role) or I(type=collection), but not both at the same run."
```

Comment:
```suggestion
    - "B(Ansible 2.9): It can only be used to install either I(type=role) or I(type=collection), but not both at the same run."
```
"a" implies (to me) that you can only install *one* role, not multiple ones.